### ISSUE TITLE
Issues with custom mpd.conf

### ISSUE THREAD

**Issue by wullewatz** [Poeschl/Hassio-Addons#80]
**Which addon?**

MPD
**Describe the bug**

- No default conf:
- Back button im ympd didnt work
- ympd config: url didnt save, always back to 127.0.0.1 (just for testing)
- conection lost after skip to next title/stop in ympd/brwser/hass mp addon/ single ympd addon
- lags in lovelace ui ### edit: just in homeassistant android app ###

I am runnig homeassistant on an raspi3
##### My MPD config: #####
@App:
shared mpd.conf / Network 6600, null / Audio default, default

@mpd.conf:
```
music_directory "/share/mpd/music"
playlist_directory "/share/mpd/playlists"
db_file "/data/database/mpd.db"
log_file		"syslog"
pid_file		"/share/mpd/mpd.pid"
state_file "/data/state"
sticker_file "/data/sticker.sql"
#user		"mpd"
#group				"audio"
#bind_to_address		"any"      <----- didn't work 4 me
port				"6600"
log_level			"verbose"
#restore_paused "no"
#save_absolute_paths_in_playlists	"no"
metadata_to_use	"artist,album,title,track,name,genre,date,composer,performer,disc"
#default_permissions             "add"      <----- didn't work 4 me
input {
        plugin "curl"
#proxy "proxy.isp.com:8080"
#proxy_user "user"
#proxy_password "password"
}
#audio_output {
#type		"alsa"
#name		"RaspiAudio"
#device		"hw:0,0"	# optional      <----- didn't work 4 me / just getted alsa worked with type&name even with group audio
#mixer_type      "hardware"	# optional      <----- 
#mixer_device	"default"	# optional      <----- 
#mixer_control	"PCM"		# optional      <----- 
#mixer_index	"0"		# optional      <----- 
#}
audio_output {
type		"fifo"
name		"mpd"
path        "/share/snapfifo/mpd"
format		"44100:24:2"
mixer_type  "null"
}
filesystem_charset		"UTF-8"
```

###### error without /share/mpd/mpd.conf in config: ######
@mpd log:
INFO: Start MPD exception: Failed to open '': No such file or directory [17:07:12] INFO: Start ympd MPD Connecting to 127.0.0.1:6600 MPD connection: Connection refused

 ###### default error message, but working: ######
@mpd log:
[s6-init] making user provided files available at /var/run/s6/etc...exited 0. [s6-init] ensuring user provided files have correct perms...exited 0. [fix-attrs.d] applying ownership & permissions fixes... [fix-attrs.d] done. [cont-init.d] executing container initialization scripts... [cont-init.d] done. [services.d] starting services [services.d] done. [17:39:02] INFO: Setup folders and settings [17:39:02] INFO: Start MPD exception: bind to '0.0.0.0:6600' failed (continuing anyway, because binding to '[::]:6600' succeeded): Failed to bind socket: Address in use path: SetFSCharset: fs charset is: libsamplerate: libsamplerate converter 'Fastest Sinc Interpolator' vorbis: Xiph.Org libVorbis 1.3.6 opus: libopus 1.3.1 hybrid_dsd: The Hybrid DSD decoder is disabled because it was not explicitly enabled simple_db: reading DB curl: version 7.67.0 curl: with OpenSSL/1.1.1g state_file: Loading state file /data/state exception: RTIOThread could not get realtime scheduling, continuing anyway: sched_setscheduler failed: Operation not permitted playlist: queue song 1:"http://br-edge-2006.fra-lg.cdn.addradio.net/br/br3/live/mp3/128/stream.mp3"

### also on invalid or non working settings in mpd.conf ###
@homeassistant log:
Protokolldetails ( ERROR )

Logger: homeassistant.components.websocket_api.http.connection.1836323280
Source: components/mpd/media_player.py:322
Integration: Home Assistant WebSocket API (documentation, issues)
Connection lost while reading lineTraceback (most recent call last): File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 130, in handle_call_service connection.context(msg), File "/usr/src/homeassistant/homeassistant/core.py", line 1253, in async_call task.result() File "/usr/src/homeassistant/homeassistant/core.py", line 1288, in _execute_service await handler.func(service_call) File "/usr/src/homeassistant/homeassistant/helpers/entity_component.py", line 213, in handle_service self._platforms.values(), func, call, required_features File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 453, in entity_service_call future.result() # pop exception if have File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 597, in async_request_call await coro File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 484, in _handle_entity_call await result File "/usr/src/homeassistant/homeassistant/components/media_player/__init__.py", line 586, in async_media_next_track await self.hass.async_add_job(self.media_next_track) File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run result = self.fn(*self.args, **self.kwargs) File "/usr/src/homeassistant/homeassistant/components/mpd/media_player.py", line 322, in media_next_track self._client.next() File "/usr/local/lib/python3.7/site-packages/mpd/base.py", line 381, in mpd_command return wrapper(self, name, args, callback) File "/usr/local/lib/python3.7/site-packages/mpd/base.py", line 473, in _execute return retval() File "/usr/local/lib/python3.7/site-packages/mpd/base.py", line 368, in command_callback res = function(self, self._read_lines()) File "/usr/local/lib/python3.7/site-packages/mpd/base.py", line 311, in _parse_nothing for line in lines: File "/usr/local/lib/python3.7/site-packages/mpd/base.py", line 538, in _read_lines line = self._read_line() File "/usr/local/lib/python3.7/site-packages/mpd/base.py", line 523, in _read_line raise ConnectionError("Connection lost while reading line") mpd.base.ConnectionError: Connection lost while reading line

@mpd log:
$next:  client: [0] command returned 0 client: [1] process command "next" playlist: play 1:"http://br-edge-2006.fra-lg.cdn.addradio.net/br/br3/live/mp3/128/stream.mp3" MPD mpd_run_status: Connection reset by peer MPD outputs: Connection reset by peer MPD connection failed.
$stop:  client: [1] process command "outputs" client: [1] command returned 0 client: [0] process command "stop" playlist: stop MPD mpd_run_status: Connection reset by peer MPD outputs: Connection reset by peer MPD connection failed.

**To Reproduce**

**Expected behavior**

**Screenshots**

**System**

* Supervisor version: 
* Host system version: 

arch | armv7l
-- | --
chassis | embedded
dev | false
docker | true
hassio | true
host_os | HassOS 3.13
installation_type | Home Assistant
os_name | Linux
os_version | 4.19.114-v7
python_version | 3.7.7
supervisor | 222
timezone | Europe/Berlin
version | 0.110.1
virtualenv | false

**Comment by wullewatz** [comment]
All errors gone with Homeassistant Update to 0.110.2 except 
### default config file.

#  Thanks a lot!

**Comment by Poeschl** [comment]
The addon will not generate a default configuration. You can use the [mpd.conf from the repository](https://github.com/Poeschl/Hassio-Addons/blob/master/mpd/mpd.conf) to start with.

**Comment by Poeschl** [comment]
Just now I realized what you meant by default config file. :sweat_smile:
will be fixed in a second.

**Comment by Poeschl** [comment]
Fixed with c5d8086
